### ISSUE TITLE
Can I use a WiFi connection？

### ISSUE THREAD

**Issue by modi12jin** [KDABLabs/flutter-remote-display#1]
Hello! For well-known reasons, classic Bluetooth is only available with esp32, so I would like to ask you if you will support WiFi? Because esp32s3 only has WiFi and BLE

**Comment by ardera** [comment]
Yep WiFi and BLE are on our TODO, especially since BLE is more suitable in this case than Bluetooth Classic, but can't give an ETA

**Comment by modi12jin** [comment]
> Yep WiFi and BLE are on our TODO, especially since BLE is more suitable in this case than Bluetooth Classic, but can't give an ETA

Which flutter package would you use for BLE? (flutter_reactive_ble or flutter_blue_plus) Do you use nimble libraries for BLE for esp32s3

**Comment by modi12jin** [comment]
For WiFi, do you use websocket or http？

**Comment by ardera** [comment]
> Which flutter package would you use for BLE? (flutter_reactive_ble or flutter_blue_plus) Do you use nimble libraries for BLE for esp32s3

Putting BLE and WiFi on our todo list is about as far as we got. Didn't look into implementation options. Any recommendations?

> For WiFi, do you use websocket or http？

Just thought going raw TCP or TLS (depending on what the ESP can do), because the current protocol we have is completely bytestream-based, so TCP would be the closest match. However could be we have to split it up into characteristics anyway to make it fit with BLE, in which case CoAP might be an option.
